### PR TITLE
Feature/testing climber

### PR DESCRIPTION
**Pull request by MajorDarius**
PTAL

### CODE REVIEW

File: src/main/cpp/commands/ClimberZeroDefaults.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#include "commands/ClimberZeroDefaults.h"
+#include "commands/SetClimber.h"
+#include "commands/MotorClimb.h"
+
+ClimberZeroDefaults::ClimberZeroDefaults() {
+  // Add Commands here:
+  // e.g. AddSequential(new Command1());
+  //      AddSequential(new Command2());
+  // these will run in order.
+  //AddSequential(new MotorClimb(0));
```
Comment (duane701) [inline src/main/cpp/commands/ClimberZeroDefaults.cpp]: Did you need to uncomment something?

File: src/main/include/commands/SetClimber.h [hunk 2]
```diff
@@ -11,10 +11,12 @@
 
 class SetClimber : public frc::Command {
  public:
-  SetClimber();
+  SetClimber(double speed);
   void Initialize() override;
   void Execute() override;
   bool IsFinished() override;
```
Comment (duane701) [inline src/main/include/commands/SetClimber.h]: Implement an `IsFinishedI()` in order to use this in a group command.

File: src/main/include/commands/MotorClimb.h [hunk 3]
```diff
@@ -11,10 +11,12 @@
 
 class MotorClimb : public frc::Command {
  public:
-  MotorClimb();
+  MotorClimb(double speed);
   void Initialize() override;
   void Execute() override;
   bool IsFinished() override;
```
Comment (duane701) [inline src/main/include/commands/MotorClimb.h]: Implement an IsFinishedI() in order to use this in a group command.

**Comment by MajorDarius** [comment]
PTAL

**Pull request merged**